guide/resources/design-research.md

```diff
@@ -97,6 +97,12 @@ A survey and follow-up interview of 990 bitcoin users to determine bitcoin manag
 
 ## Independent research
 
+#### [2024 American Bitcoin Survey report](https://www.thenakamotoproject.org/report)
```

Comment:
Should we link to the actual report link itself? Not sure if this is correct though...
```suggestion
#### [2024 American Bitcoin Survey report](https://cdn.prod.website-files.com/6644db90e58a1fbeb1c70ddc/669f69702fa1b4b3bf09dbac_TNP%20Q3%202024%20Report%20(Digital).pdf)
```

Reply:
I understand it's nice to go around their email form, but I would honor how their decisions and keep the link to the report landing page.